### DISCUSSION TITLE
Arduino GIGA R1 WiFi

### DISCUSSION THREAD

**Discussion by attreewr** [board707/DMD_STM32#61] [Q&A]
Hi, are there any ports available that support the Arduino GIGA R1 WiFi  ?

**Comment by board707** [comment]
Thank you for your attention.
No, there is no port for Arduino Giga.
In principle, since Giga is also a STM32, it is possible to adapt the library, but so far I haven’t even held Giga in my hands, so this is not a matter of the near future.
